cmd/kops/replace.go

```diff
@@ -123,16 +132,32 @@ func RunReplace(f *util.Factory, cmd *cobra.Command, out io.Writer, c *ReplaceOp
 				if cluster == nil {
 					return fmt.Errorf("cluster %q not found", clusterName)
 				}
-				_, err = clientset.InstanceGroupsFor(cluster).Update(v)
+				// @check if the instancegroup exists already
+				igName := v.ObjectMeta.Name
+				ig, err := clientset.InstanceGroupsFor(cluster).Get(igName, metav1.GetOptions{})
 				if err != nil {
-					return fmt.Errorf("error replacing instanceGroup: %v", err)
+					return fmt.Errorf("unable to check for instanceGroup: %v", err)
```

Comment:
Interesting... I'm surprised this doesn't return an error where `errors.IsNotFound(err) == true`.